test/test_sparse.py

```diff
@@ -3251,26 +3251,49 @@ class TestSparseUnaryUfuncs(TestCase):
 
     @ops(sparse_unary_ufuncs)
     def test_sparse_consistency(self, device, dtype, op):
-        unsupportedTypes = [torch.bfloat16, torch.cfloat, torch.cdouble]
+        unsupportedTypes = [torch.bfloat16, torch.cfloat, torch.cdouble, torch.bool]
         if dtype in unsupportedTypes:
             self.skipTest('Skipped! Unsupported dtypes for Sparse')
 
-        samples = op.sample_inputs(device, dtype)
+        samples = op.sample_sparse_inputs(device, dtype)
 
         if len(samples) == 0:
             self.skipTest("Skipped! No sample inputs!")
 
-        sample = samples[0]
+        for sample in samples:
+            sample = sample.input[0]
+            expected = op(sample.to_dense())
+            output = op(sample)
+            self.assertEqual(output.to_dense(), expected)
 
-        if len(sample.input) > 1:
-            self.skipTest("Skipped! Testing unary ops, one input is expected")
-        sample = sample.input[0]
+    @ops(sparse_unary_ufuncs)
+    def test_sparse_inplace_consistency(self, device, dtype, op):
+        unsupportedTypes = [torch.bfloat16, torch.cfloat, torch.cdouble, torch.bool]
+        if dtype in unsupportedTypes:
+            self.skipTest('Skipped! Unsupported dtypes for Sparse')
+
+        inplace_variant = op.get_inplace()
 
-        expected = op(sample)
-        assert torch.is_tensor(expected)
-        output = op(sample.to_sparse())
-        assert torch.is_tensor(output)
-        self.assertEqual(output.to_dense(), expected)
+        if inplace_variant is not None:
+            samples = op.sample_sparse_inputs(device, dtype)
+            if len(samples) == 0:
+                self.skipTest("Skipped! No sample inputs!")
+
+            for sample in samples:
+                sample = sample.input[0]
```

Comment:
This is OK for now but why we need a TODO. If the input had multiple tensors (like a binary operation, like add, for example) then only the first one would be made dense here. In the future we can extend this test to handle more function types. We don't need to do it in this PR.

Reply:
Put in a TODO as suggested